models/public/mozilla-deepspeech-0.8.2/mozilla-deepspeech-0.8.2.md

```diff
@@ -0,0 +1,138 @@
+
+For details on this model, see <https://github.com/mozilla/DeepSpeech/releases/tag/v0.8.2>.
+
+## Conversion
+
+1. Download `deepspeech-0.8.2-models.pbmm` and `deepspeech-0.8.2-models.scorer` files from Mozilla's repository (MPL-2.0 license):
+   ```
+   source <openvino_path>/bin/setupvars.sh
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/downloader.py --name mozilla-deepspeech-0.8.2
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mozilla-deepspeech-0.8.2
+   ```
+1. Install TensorFlow v1 (tested with v1.15.3) and protobuf (tested with 3.13.0):
+   ```
+   python3 -m pip install 'tensorflow>=1.0,<2.0' 'protobuf>=3.13.0,<4.0'
+   ```
+1. Run conversion from .pbmm ("memory-mapped" graph) to .pb (the common GraphDef):
+   ```
+   ./pbmm_to_pb.py <download_path>/deepspeech-0.8.2-models.pbmm <download_path>/deepspeech-0.8.2-models.pb
+   ```
+1. Run Model Converter to convert .pb to IR files:
+   ```
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mozilla-deepspeech-0.8.2
+   ```
+1. Run conversion from .scorer (language model format in Mozilla DeepSpeech v0.7.x/0.8.x) to .kenlm (LM format used in v0.6.x and by us):
+   ```
+   ./scorer_to_kenlm.py <download_path>/deepspeech-0.8.2-models.scorer <download_path>/deepspeech-0.8.2-models.kenlm
+   ```
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Speech recognition                        |
+| GFlops per audio frame          | 0.0472                                    |
+| GFlops per second of audio      | 2.36                                      |
+| MParams                         | 47.2                                      |
+| Source framework                | TensorFlow\*                              |
+
+## Accuracy
+
+| Metric                       | Value      | Parameters                                                       |
+| ---------------------------- | ---------- | ---------------------------------------------------------------- |
+| WER @ LibriSpeech test-clean | 8.39%      | with LM, beam_width = 32, Python CTC decoder, accuracy checker   |
+| WER @ LibriSpeech test-clean | **6.13%**  | with LM, **beam_width = 500**, C++ CTC decoder, accuracy checker |
+| WER @ LibriSpeech test-clean | **6.15%**  | with LM, **beam_width = 500**, C++ CTC decoder, demo             |
+
+*NB*: beam_width=32 is a low value for a CTC decoder, and was used to achieve reasonable evaluation time with Python CTC decoder in Accuracy Checker.
+Increasing beam_width improves WER metric and slows down decoding.  Speech recognition demo has a faster C++ CTC decoder module.
+
+## Performance
+
+## Input
+
+### Original Model
+
+ * **Audio MFCC coefficients**, name: `input_node` , shape: [1x16x19x26], format: [BxNxTxC], where:
```

Comment:
Please, use numbered list instead of bullet list.

Reply:
Done.